QandA/08-conditions.Rmd

```diff
@@ -8,20 +8,16 @@ library(rlang)
 ## 8.3 Ignoring conditions {-}
 
 :::question
-I can't quite think of a time where `try` is more appropraite than `tryCatch` - does anyone have an example?
+I can't quite think of a time where `try` is more appropriate than `tryCatch` - does anyone have an example?
 :::
 
-:::TODO
-XXX
-:::
+It seems that try is just a wrapper for `tryCatch` and you can use it whenever you'd use `try`
```

Comment:
From https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/try 
> try is implemented using tryCatch; for programming, instead of try(expr, silent = TRUE), something like tryCatch(expr, error = function(e) e) (or other simple error handler functions) may be more efficient and flexible.

My understanding is essentially - try is a tryCatch with "return NULL" as the default error handler